.github/workflows/rust.yml

```diff
@@ -84,6 +80,14 @@ jobs:
     - name: Compile the workspace with the default features (build)
       run: |
         cargo build --locked
+    - name: Run Storage Service
+      run: |
+        ./target/debug/storage_service_server memory --endpoint 127.0.0.1:1235 &
+        cargo test --features storage_service -- storage_service
```

Comment:
Please make the endpoint `127.0.0.1:1235` available in the environment using something like:
`echo "LINERA_STORAGE_SERVICE=127.0.0.1:1235" >> "$GITHUB_ENV"`

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
I used that solution.